tests/integration/tools/test_composio.py

```diff
@@ -0,0 +1,33 @@
+import json
+import os
+
+from any_agent import AgentConfig, AgentFramework, AnyAgent
+from any_agent.tracing.attributes import GenAI
+from any_agent.tools.composio import CallableProvider
+
+from composio import Composio
+
+
+def test_composio(agent_framework: AgentFramework) -> None:
+    cpo = Composio(CallableProvider())
+    tools = cpo.tools.get(
+        user_id=os.environ["COMPOSIO_USER_ID"],
+        search="repository issues",
+        toolkits=["GITHUB"],
+        limit=10,
+    )
+    agent = AnyAgent.create(
```

Comment:
Can you help me understand this: what does the .get call return, a dict or a callable? 

I liked the original design where this gives us back a json schema, because that is what lets us serialize it and save it in the any-forge state json file. 

I assume your design returns a callable since there were no edits made to the typing of AgentConfig.tools.